### PR TITLE
Update IoT Central Guidance

### PR DESCRIPTION
**Pull request by randywu763**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by daisukeiot — APPROVED** [review]

**Review by daisukeiot — APPROVED** [review]

**Review by daisukeiot — APPROVED** [review]

**Review by daisukeiot — APPROVED** [review]

File: Dev_Tools_Install.md [hunk 1]
```diff
@@ -18,19 +18,9 @@ Please install the following software in the exact order listed:
 
     <img src=".//media/image15.png"/>
 
-3. Download and run Microchip's [IoT Provisioning Tool](https://github.com/microchip-pic-avr-tools/iotprovision-bin/releases/latest) which is used to (1) generate the security certificates used for authentication and (2) store them into the [ATWINC1510](https://www.microchip.com/en-us/product/ATWINC1510) Wi-Fi controller module by executing the following steps:
+3. Install the Microsoft [Azure Command Line Interface (CLI)](https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest). The Azure CLI is a set of commands used to create and manage Azure resources. The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation
```
Comment (daisukeiot) [inline Dev_Tools_Install.md:23]: Remove language field from the link

https://docs.microsoft.com/en-us/cli/azure/?view=azure-cli-latest to

https://docs.microsoft.com/cli/azure/?view=azure-cli-latest

**Review by daisukeiot — APPROVED** [review]

File: SAM_IoT_Central.md [hunk 2]
```diff
@@ -158,33 +165,37 @@ IoT Central allows you to create an application dashboard to monitor the telemet
...
 
     <img src=".//media/image88.png" style="width:5.in;height:1.58982in" alt="A screenshot of a cell phone Description automatically generated" />
 
-14. Within 5 seconds of sending the Reboot command, the SAM-IoT development board should reset itself.  Once the Blue and Green LED's stay constantly ON, press the SW0 and SW1 buttons (the Red LED may toggle with each button press)
+14. Within 5 seconds of sending the Reboot command, the SAM-IoT development board should reset itself.  Once the Blue and Green LED's **both** stay constantly ON, press the SW0 and SW1 buttons (the Red LED may toggle with each button press)
 
     <img src=".//media/image89.png" style="width:5.in;height:1.28982in" alt="A screenshot of a cell phone Description automatically generated" />
 
-15. Click on the `Raw data` tab and confirm that the button press telemetry messages were received
+15. Click on the `Properties (Writable)` tab and type `0` (zero) in the `SW0/SW1 button push event` field, then hit `Save`
```
[the remark this answers is not in the document]
Comment (randywu763) [reply SAM_IoT_Central.md:184]: Correct - should be "Disable Telemetry" field

File: Dev_Tools_Install.md (hunk 1 above)
Comment (randywu763) [reply Dev_Tools_Install.md:23]: Done

File: SAM_IoT_Hub.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ Upload your verification certificate to DPS for [Proof of Possession](https://t
 2. Change the directory to your certificates folder (which was generated by the [IoT Provisioning Tool](https://www.microchip.com/en-us/solutions/internet-of-things/iot-development-kits/iot-provisioning-tool)):
 
     ```bash
-    cd drive\[your path\.microchip-iot
+    cd drive\[your path]\.microchip-iot
```
[the remark this answers is not in the document]
Comment (randywu763) [reply SAM_IoT_Hub.md:50]: Changed to

cd drive:\<your_path>\.microchip-iot

**Pull request merged**